content/development/how-to-write-a-loader.md

```diff
@@ -1,5 +1,7 @@
 ---
 title: How to write a loader?
+contributors:
```

Comment:
Minor nitpick: can you move the the `contributors` under `sort`? It's nice to see the sort order and title easily